paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -717,13 +709,18 @@ internal class CustomerSheetViewModel(
                         enabled = false,
                     )
                 }
-                val formFieldValues = currentViewState.formFieldValues ?: error("completeFormValues cannot be null")
-                val params = formFieldValues
-                    .transformToPaymentMethodCreateParams(
+
+                val createParams = if (currentViewState.bankAccountSelection != null) {
```

Comment:
The US Bank Account flow is now also handled here, albeit as a special case compared to cards.